pbr/pbr.py

```diff
@@ -71,6 +72,15 @@ def main():
         for ii in range(scene_config.num_robots + 1)
     ]
 
+    misc_robots = [
```

Comment:
Apparently it is common to have multiple kinds of robots on the same field. 

The preferred behaviour for this is now to choose a random robot each time a robot is added to scene.

Reply:
I'm pretty sure that's how it works right now. 
![NUpbr_Example](https://github.com/NUbots/NUpbr/assets/91118492/efbfc77e-0abc-486d-90e0-60860891a890)

Reply:
I've had the same outputs as Joe, where there are different misc robots generating in the same scene